Any keystore except standard manager keystore causing manager to crash

### Bug description

I have tried to patch YouTube 18.32.39 with the latest Revanced manager using revancify keystore but the manager crashes on exporting apk. And the exported apk file size is zero bits(corrupted). I thought that the problem is due to revancify keystore but i have also tried to use my custom keystore and Revanced keystore generated when patching using CLI on pc but the problem still exists.

https://github.com/ReVanced/revanced-manager/assets/107951133/9b78349d-aa84-4bb2-b681-eee486b4860f





### Version of ReVanced Manager and version & name of application you tried to patch

ReVanced manager 1.9.5
YouTube 18.32.39

### Installation type

Non-root

### Device logs

```shell
--------- beginning of crash
09-09 12:46:28.355 22727 22727 E AndroidRuntime: FATAL EXCEPTION: main
09-09 12:46:28.355 22727 22727 E AndroidRuntime: Process: app.revanced.manager.flutter, PID: 22727
09-09 12:46:28.355 22727 22727 E AndroidRuntime: java.io.FileNotFoundException: /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-CCEOIX/out.apk: open failed: ENOENT (No such file or directory)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at libcore.io.IoBridge.open(IoBridge.java:496)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at java.io.FileInputStream.<init>(FileInputStream.java:159)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at com.cleveroad.cr_file_saver.utils.FileKt$saveFileInBackground$1$file$1.invokeSuspend(File.kt:71)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:100)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@3e6aef6, Dispatchers.Main]
09-09 12:46:28.355 22727 22727 E AndroidRuntime: Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at libcore.io.Linux.open(Native Method)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:252)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:8472)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	at libcore.io.IoBridge.open(IoBridge.java:482)
09-09 12:46:28.355 22727 22727 E AndroidRuntime: 	... 10 more
09-09 13:24:43.342  1220  1220 E AndroidRuntime: FATAL EXCEPTION: main
09-09 13:24:43.342  1220  1220 E AndroidRuntime: Process: app.revanced.manager.flutter, PID: 1220
09-09 13:24:43.342  1220  1220 E AndroidRuntime: java.io.FileNotFoundException: /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-HUKTXV/out.apk: open failed: ENOENT (No such file or directory)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at libcore.io.IoBridge.open(IoBridge.java:496)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at java.io.FileInputStream.<init>(FileInputStream.java:159)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at com.cleveroad.cr_file_saver.utils.FileKt$saveFileInBackground$1$file$1.invokeSuspend(File.kt:71)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:100)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@685201b, Dispatchers.Main]
09-09 13:24:43.342  1220  1220 E AndroidRuntime: Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at libcore.io.Linux.open(Native Method)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:252)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:8472)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	at libcore.io.IoBridge.open(IoBridge.java:482)
09-09 13:24:43.342  1220  1220 E AndroidRuntime: 	... 10 more
--------- beginning of system
09-09 13:26:40.063  5280  5280 I ViewRootImpl: dispatchDetachedFromWindow in doDie
09-09 13:26:43.593  5280  5280 I ViewRootImpl: dispatchDetachedFromWindow in doDie
09-09 13:26:43.842  5280  5280 D ActivityThread: Remove activity client record, r= ActivityRecord{ba8d741 token=android.os.BinderProxy@a0a92aa {app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity}} token= android.os.BinderProxy@a0a92aa
09-09 13:26:43.842  5280  5280 I ViewRootImpl: dispatchDetachedFromWindow in doDie
09-09 13:26:47.117  5280  5280 I ViewRootImpl: dispatchDetachedFromWindow in doDie
09-09 13:29:37.396 10011 10011 D ActivityThread: Attach thread to application
09-09 13:29:37.404 10011 11459 I HwActivityThreadImpl: mSocketName: d36b5aeb-c3dd-414c-991e-b30bcb2b9e55
09-09 13:29:37.405 10011 11459 I HwActivityThreadImpl: initPreloadedSocked end mPreloadStatus 2
09-09 13:29:37.408 10011 11459 I HwActivityThreadImpl: before connect app.socketName:d36b5aeb-c3dd-414c-991e-b30bcb2b9e55
09-09 13:29:37.427 10011 11461 I ActivityThread: Init compatible state: true
09-09 13:29:37.475 10011 10011 I HwActivityThreadImpl: accept socket mSocketName: d36b5aeb-c3dd-414c-991e-b30bcb2b9e55
09-09 13:29:37.475 10011 10011 I HwActivityThreadImpl: accept socket end
09-09 13:29:37.475 10011 10011 I HwActivityThreadImpl: acceptPreloadedApplication end mPreloadStatus 3
09-09 13:29:37.475 10011 11459 I HwActivityThreadImpl: connectContinuePreload end mPreloadStatus 3
09-09 13:29:37.492 10011 10011 I ActivityThread: finishPreloaded preloadStatus 3
09-09 13:29:37.492 10011 10011 I ActivityThread: finishPreloaded end preloadStatus 3
09-09 13:29:37.493 10011 10011 I RtgSchedEvent: current pid:10011 AppType:-1
09-09 13:29:37.525 10011 10011 V ActivityThread: callActivityOnCreate
09-09 13:29:37.635 10011 10011 D ActivityThread: add activity client record, r= ActivityRecord{ba8d741 token=android.os.BinderProxy@a0a92aa {app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity}} token= android.os.BinderProxy@a0a92aa
09-09 13:29:37.654 10011 10011 I RtgSchedEvent: current pid:10011 AppType:-1
09-09 13:29:41.984 10011 10011 W HwApsManager: HwApsManagerService, registerCallback, start !
09-09 13:29:46.664 10011 10011 D ActivityThread: Won't deliver top position change in state=5
09-09 13:29:46.674 10011 10011 I RtgSchedEvent: current pid:10011 AppType:-1
09-09 13:29:48.061 10011 10011 W PackageParser: Unknown element under <manifest>: queries at /data/user/0/app.revanced.manager.flutter/cache/file_picker/YouTube_18.32.39_apkcombo.com.apk Binary XML file line #41
09-09 13:29:48.061 10011 10011 W PackageParser: Unknown element under <application>: property at /data/user/0/app.revanced.manager.flutter/cache/file_picker/YouTube_18.32.39_apkcombo.com.apk Binary XML file line #418
09-09 13:29:48.082 10011 10011 D SystemConfig: Removed unavailable feature android.hardware.sensor.barometer
09-09 13:29:48.082 10011 10011 D SystemConfig: Removed unavailable feature android.hardware.sensor.gyroscope
09-09 13:29:48.083 10011 10011 W SystemConfig: No directory /product_services/etc/sysconfig, skipping
09-09 13:29:48.083 10011 10011 W SystemConfig: No directory /product_services/etc/permissions, skipping
09-09 13:29:48.085 10011 10011 W SystemConfig: No directory /system/emui/base/sysconfig, skipping
09-09 13:29:48.086 10011 10011 W SystemConfig: No directory /system/emui/oversea/sysconfig, skipping
09-09 13:29:48.086 10011 10011 W SystemConfig: No directory /vendor/etc/sysconfig, skipping
09-09 13:29:48.088 10011 10011 D SystemConfig: Removed unavailable feature android.hardware.sensor.barometer
09-09 13:29:48.088 10011 10011 D SystemConfig: Removed unavailable feature android.hardware.sensor.gyroscope
09-09 13:29:48.090 10011 10011 W SystemConfig: No directory /odm/etc/sysconfig, skipping
09-09 13:29:48.091 10011 10011 W SystemConfig: No directory /odm/etc/carrier/60203/sysconfig, skipping
09-09 13:29:48.091 10011 10011 W SystemConfig: No directory /odm/etc/carrier/60203/permissions, skipping
09-09 13:29:48.091 10011 10011 W SystemConfig: No directory /odm/hw_odm/JNY-L21/sysconfig, skipping
09-09 13:29:48.092 10011 10011 W SystemConfig: No directory /odm/hw_odm/JNY-L21/permissions, skipping
09-09 13:29:48.092 10011 10011 W SystemConfig: No directory /hw_product/etc/sysconfig, skipping
09-09 13:29:48.093 10011 10011 W SystemConfig: No directory /hw_product/etc/carrier/60203/sysconfig, skipping
09-09 13:29:48.093 10011 10011 W SystemConfig: No directory /hw_product/etc/carrier/60203/permissions, skipping
09-09 13:29:48.093 10011 10011 W SystemConfig: No directory /hw_product/region_comm/oversea/sysconfig, skipping
09-09 13:29:48.094 10011 10011 W SystemConfig: No directory /hw_product/hw_oem/JNY-L21/sysconfig, skipping
09-09 13:29:48.094 10011 10011 W SystemConfig: No directory /preload/JNY-L21/hw/meafnaf/sysconfig, skipping
09-09 13:29:48.094 10011 10011 W SystemConfig: No directory /cust/global/sysconfig, skipping
09-09 13:29:48.094 10011 10011 W SystemConfig: No directory /cust/global/permissions, skipping
09-09 13:29:48.094 10011 10011 W SystemConfig: No directory /cust/global/carrier/60203/sysconfig, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /cust/global/carrier/60203/permissions, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /cust/hw/meafnaf/sysconfig, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /cust/hw/meafnaf/carrier/60203/sysconfig, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /cust/hw/meafnaf/carrier/60203/permissions, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /hw_product/special_cust/JNY-L21/hw/meafnaf/sysconfig, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /hw_product/special_cust/JNY-L21/hw/meafnaf/permissions, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /version/special_cust/JNY-L21/hw/meafnaf/sysconfig, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /version/special_cust/JNY-L21/hw/meafnaf/permissions, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /cust/cotalite/sysconfig, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /cust/cotalite/permissions, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /cust/ecota/sysconfig, skipping
09-09 13:29:48.095 10011 10011 W SystemConfig: No directory /cust/ecota/permissions, skipping
09-09 13:29:51.048 10011 10011 D HwCustConnectivityManagerImpl: isBlockNetworkRequestByNonAis, INVALID_SUBSCRIPTION_ID
09-09 13:33:02.671 10011 10011 I RtgSchedEvent: current pid:10011 AppType:-1
--------- beginning of main
09-09 13:33:41.059 10011 12502 W System.err: java.security.UnrecoverableKeyException: no match
09-09 13:33:41.059 10011 12502 W System.err: 	at com.android.org.bouncycastle.jcajce.provider.keystore.bc.BcKeyStoreSpi$StoreEntry.getObject(BcKeyStoreSpi.java:325)
09-09 13:33:41.059 10011 12502 W System.err: 	at com.android.org.bouncycastle.jcajce.provider.keystore.bc.BcKeyStoreSpi.engineGetKey(BcKeyStoreSpi.java:620)
09-09 13:33:41.059 10011 12502 W System.err: 	at java.security.KeyStore.getKey(KeyStore.java:1062)
09-09 13:33:41.059 10011 12502 W System.err: 	at app.revanced.manager.flutter.utils.signing.Signer.signApk(Signer.kt:63)
09-09 13:33:41.059 10011 12502 W System.err: 	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$13(MainActivity.kt:279)
09-09 13:33:41.059 10011 12502 W System.err: 	at app.revanced.manager.flutter.MainActivity.$r8$lambda$6dokpIuPm1kqn7qLA9JtiVzSCi0(Unknown Source:0)
09-09 13:33:41.059 10011 12502 W System.err: 	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda3.run(Unknown Source:24)
09-09 13:33:41.059 10011 12502 W System.err: 	at java.lang.Thread.run(Thread.java:929)
09-09 13:33:41.194 10011 11454 W System  : A resource failed to call close. 
09-09 13:33:43.491 10011 10011 V AudioManager: querySoundEffectsEnabled...
09-09 13:33:43.533 10011 10011 D DecorView: showOrHideHighlightView: hasFocus=false; winMode=1; isMrgNull=true
09-09 13:33:43.533 10011 10011 W HwRemoteInputMethodManager: isCasting false because IHwDistributedWindowManager is invalid.
09-09 13:33:43.712 10011 11468 D ZrHung.AppEyeUiProbe: not watching, wait.
09-09 13:33:43.732 10011 10011 I RmeSchedManager: init Rme, version is: v1.0
09-09 13:33:44.012 10011 11481 W libEGL  : EGLNativeWindowType 0x77f1497010 disconnect failed
09-09 13:33:44.016 10011 11474 W libEGL  : EGLNativeWindowType 0x797860ea50 disconnect failed
09-09 13:33:45.198 10011 10011 D ActivityThread: Won't deliver top position change in state=5
09-09 13:33:45.198 10011 10011 I InstallPlugin: handleActivityResult(2,-1,Intent { dat=content://com.android.providers.downloads.documents/document/1557 flg=0x43 })
09-09 13:33:45.206 10011 11468 D ZrHung.AppEyeUiProbe: restart watching
09-09 13:33:45.209 10011 10011 I RmeSchedManager: init Rme, version is: v1.0
09-09 13:33:45.209 10011 10011 I RtgSchedEvent: current pid:10011 AppType:-1
09-09 13:33:45.218 10011 11474 D mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
09-09 13:33:45.220 10011 11481 D mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
09-09 13:33:45.249 10011 10011 I HwViewRootImpl: removeInvalidNode all the node in jank list is out of time
09-09 13:33:45.253 10011 10011 E AndroidRuntime: FATAL EXCEPTION: main
09-09 13:33:45.253 10011 10011 E AndroidRuntime: Process: app.revanced.manager.flutter, PID: 10011
09-09 13:33:45.253 10011 10011 E AndroidRuntime: java.io.FileNotFoundException: /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-AXYSFX/out.apk: open failed: ENOENT (No such file or directory)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at libcore.io.IoBridge.open(IoBridge.java:496)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at java.io.FileInputStream.<init>(FileInputStream.java:159)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at com.cleveroad.cr_file_saver.utils.FileKt$saveFileInBackground$1$file$1.invokeSuspend(File.kt:71)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:100)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@bc41a36, Dispatchers.Main]
09-09 13:33:45.253 10011 10011 E AndroidRuntime: Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at libcore.io.Linux.open(Native Method)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:252)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:8472)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	at libcore.io.IoBridge.open(IoBridge.java:482)
09-09 13:33:45.253 10011 10011 E AndroidRuntime: 	... 10 more
09-09 13:33:45.265 10011 10011 I Process : Sending signal. PID: 10011 SIG: 9
09-09 13:33:55.451 13025 13025 E manager.flutte: LoadAppImageStartupCache enabled : 1
09-09 13:33:55.453 13025 13025 I manager.flutte: Reinit property: dalvik.vm.checkjni= false
09-09 13:33:55.453 13025 13025 I manager.flutte: ReInitProperties: persist.vm.debug.dumpapi= false
09-09 13:33:55.454 13025 13025 E manager.flutte: Not starting debugger since process cannot load the jdwp agent.
09-09 13:33:55.463 13025 13025 D ActivityThread: Attach thread to application
09-09 13:33:55.469 13025 13463 I HwActivityThreadImpl: mSocketName: 95d99bd5-5154-4c2b-aa9d-e992e7f67709
09-09 13:33:55.470 13025 13463 I HwActivityThreadImpl: initPreloadedSocked end mPreloadStatus 2
09-09 13:33:55.475 13025 13462 I ActivityThread: Init compatible state: true
09-09 13:33:55.476 13025 13462 D HwFrameworkSecurityPartsFactory: HwFrameworkSecurityPartsFactory in.
09-09 13:33:55.476 13025 13462 I HwFrameworkSecurityPartsFactory: add HwFrameworkSecurityPartsFactory to memory.
09-09 13:33:55.477 13025 13462 E BehaviorCollectManager: Fail to acquire dataAnalyzerService...
09-09 13:33:55.493 13025 13025 D ApplicationLoaders: createClassLoader zip: /data/app/app.revanced.manager.flutter-tP-XDvRTc5jJPROZqaCrJA==/base.apk librarySearchPath: /data/app/app.revanced.manager.flutter-tP-XDvRTc5jJPROZqaCrJA==/lib/arm64:/data/app/app.revanced.manager.flutter-tP-XDvRTc5jJPROZqaCrJA==/base.apk!/lib/arm64-v8a libraryPermittedPath: /data/user/0/app.revanced.manager.flutter parent: java.lang.BootClassLoader@dde43f5 targetSdkVersion: 33 isBundled: false classLoaderName: null sharedLibraries: null
09-09 13:33:55.513 13025 13462 I HwApiCacheMangerEx: apicache path=/storage/emulated/0 state=mounted key=app.revanced.manager.flutter#12465#256
09-09 13:33:55.513 13025 13462 I HwApiCacheMangerEx: need clear apicache,because volumes changed,oldCnt=0 newCnt=1
09-09 13:33:55.513 13025 13462 I HwApiCacheMangerEx: apicache path=/storage/emulated/0 state=mounted key=app.revanced.manager.flutter#12465#0
09-09 13:33:55.513 13025 13462 I HwApiCacheMangerEx: need clear apicache,because volumes changed,oldCnt=1 newCnt=1
09-09 13:33:55.514 13025 13464 I HwActivityThreadImpl: before connect app.socketName:95d99bd5-5154-4c2b-aa9d-e992e7f67709
09-09 13:33:55.515 13025 13025 I HwActivityThreadImpl: accept socket mSocketName: 95d99bd5-5154-4c2b-aa9d-e992e7f67709
09-09 13:33:55.515 13025 13025 I HwActivityThreadImpl: accept socket end
09-09 13:33:55.516 13025 13464 W HwActivityThreadImpl: closeStream error.
09-09 13:33:55.516 13025 13464 I HwActivityThreadImpl: connectContinuePreload end mPreloadStatus 3
09-09 13:33:55.516 13025 13025 W HwActivityThreadImpl: closeStream error.
09-09 13:33:55.516 13025 13025 I HwActivityThreadImpl: acceptPreloadedApplication end mPreloadStatus 3
09-09 13:33:55.522 13025 13462 I HwPartIawareFactory: add android.common.HwPartIawareFactoryImpl to memory.
09-09 13:33:55.522 13025 13462 I AwareBitmapCacher: init processName:app.revanced.manager.flutter pid=13025 uid=12465
09-09 13:33:55.522 13025 13025 I HwApiCacheMangerEx: apicache path=/storage/emulated/0 state=mounted key=app.revanced.manager.flutter#12465#256
09-09 13:33:55.522 13025 13025 I HwApiCacheMangerEx: need clear apicache,because volumes changed,oldCnt=1 newCnt=1
09-09 13:33:55.528 13025 13474 E AwareLog: AtomicFileUtils: readFileLines file not exist: android.util.AtomicFile@2cc0d9
09-09 13:33:55.535 13025 13025 I HwTypeface: updateFont: scale=1.0
09-09 13:33:55.539 13025 13025 I FontFamily: buildHwFonts: add Font {nativePtr=521535021344, path=/data/skin/fonts/DroidSansChinese.ttf, style=FontStyle { weight=400, slant=0}, ttcIndex=0, axes=, localeList=, buffer=java.nio.DirectByteBuffer[pos=0 lim=49724 cap=49724], isFontWeightVariable=false} to [Font {nativePtr=521535021344, path=/data/skin/fonts/DroidSansChinese.ttf, style=FontStyle { weight=400, slant=0}, ttcIndex=0, axes=, localeList=, buffer=java.nio.DirectByteBuffer[pos=0 lim=49724 cap=49724], isFontWeightVariable=false}]
09-09 13:33:55.539 13025 13025 D SystemFonts: isUsingHwThemeFonts: online invariable font existed
09-09 13:33:55.539 13025 13025 I ActivityThread: finishPreloaded preloadStatus 3
09-09 13:33:55.550 13025 13025 I ActivityThread: finishPreloaded end preloadStatus 3
09-09 13:33:55.550 13025 13025 I RmeSchedManager: init Rme, version is: v1.0
09-09 13:33:55.550 13025 13025 I RmeSchedManager: init Rme, version is: v1.0
09-09 13:33:55.550 13025 13025 I RtgSchedEvent: current pid:13025 AppType:-1
09-09 13:33:55.551 13025 13025 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@a0a92aa
09-09 13:33:55.558 13025 13475 I OpenGLRenderer: RenderThread: progress name = [app.revanced.manager.flutter], viewCornerType = 0
09-09 13:33:55.560 13025 13477 I iGraphics: [0020080c] pn: app.revanced.manager.flutter, p: 13025
09-09 13:33:55.560 13025 13477 I iGraphics: [0030080c] no spt app: app.revanced.manager.flutter
09-09 13:33:55.568 13025 13025 V ActivityThread: callActivityOnCreate
09-09 13:33:55.622 13025 13025 W manager.flutte: Accessing hidden method Landroid/view/WindowInsets;->getInsets(I)Landroid/graphics/Insets; (blacklist, linking, denied)
09-09 13:33:55.622 13025 13025 I chatty  : uid=12465(app.revanced.manager.flutter) identical 1 line
09-09 13:33:55.622 13025 13025 W manager.flutte: Accessing hidden method Landroid/view/WindowInsets;->getInsets(I)Landroid/graphics/Insets; (blacklist, linking, denied)
09-09 13:33:55.635 13025 13025 I DecorView[]: pkgName:app.revanced.manager.flutter old windowMode:0 new windoMode:1, isFixedSize:false
09-09 13:33:55.638 13025 13025 D ActivityThread: add activity client record, r= ActivityRecord{ba8d741 token=android.os.BinderProxy@a0a92aa {app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity}} token= android.os.BinderProxy@a0a92aa
09-09 13:33:55.645 13025 13025 D MouseWheelSynthesizer: mMoveStepInDp: 64, mMoveStepInPixel: 192, mUpTimeDelayed: 100
09-09 13:33:55.646 13025 13025 D ViewRootImpl: ViewRootImpl mIsInProductivePCDisplay: false
09-09 13:33:55.646 13025 13025 I HwPartPowerOfficeFactory: add HwPartPowerOfficeFactoryImpl to memory.
09-09 13:33:55.649 13025 13025 I DecorView[]: pkgName:app.revanced.manager.flutter old windowMode:1 new windoMode:1, isFixedSize:false
09-09 13:33:55.652 13025 13025 D InputEventReceiver: dispatchInputInterval 1000000
09-09 13:33:55.653 13025 13025 I RmeSchedManager: init Rme, version is: v1.0
09-09 13:33:55.653 13025 13025 I RtgSchedEvent: current pid:13025 AppType:-1
09-09 13:33:55.659 13025 13506 D HiTouch_PressGestureDetector: onAttached, package=app.revanced.manager.flutter, windowType=1, mIsHiTouchRestricted=false
09-09 13:33:55.685 13025 13025 D RenderService: RCS is disable
09-09 13:33:55.687 13025 13481 D mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
09-09 13:33:55.688 13025 13475 D mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
09-09 13:33:56.075 13025 13025 D DecorView: showOrHideHighlightView: hasFocus=true; winMode=1; isMrgNull=true
09-09 13:33:56.075 13025 13025 W InputMethodManager: startInputReason = 1
09-09 13:33:56.075 13025 13025 W HwRemoteInputMethodManager: isCasting false because IHwDistributedWindowManager is invalid.
09-09 13:33:56.078 13025 13025 W InputMethodManager: startInputReason = 5
09-09 13:33:56.087 13025 13480 I flutter : dynamic_color: Dynamic color not detected on this device.
09-09 13:33:56.107 13025 13025 D permissions_handler: No permissions found in manifest for: []22
09-09 13:33:56.118 13025 13025 I DecorView[]: pkgName:app.revanced.manager.flutter old windowMode:1 new windoMode:1, isFixedSize:false
09-09 13:33:56.124 13025 13481 W Gralloc3: mapper 3.x is not supported
09-09 13:33:56.156 13025 13025 I HwViewRootImpl: removeInvalidNode jank list is null
09-09 13:33:57.035 13025 13025 I Hwaps   : APS: EventAnalyzed: initAPS: version is 11.0.0.4
09-09 13:33:57.035 13025 13025 D Hwaps   : Fpsrequest create,type:EXACTLY_IDENTIFY
09-09 13:33:57.035 13025 13025 D Hwaps   : Fpsrequest create,type:EXACTLY_IDENTIFY
09-09 13:33:57.035 13025 13025 D Hwaps   : Fpsrequest create,type:OPENGL_SETTING
09-09 13:33:57.035 13025 13025 D Hwaps   : FpsController create
09-09 13:33:57.037 13025 13025 D Hwaps   : APS: EventAnalyzed: reInitFpsPara :mBaseFps = 60; mMaxFps = 60
09-09 13:33:57.037 13025 13025 W Settings: Setting device_provisioned has moved from android.provider.Settings.Secure to android.provider.Settings.Global.
09-09 13:33:57.037 13025 13025 V HiTouch_HiTouchSensor: User setup is finished.
09-09 13:33:57.136 13025 13025 W HwApsManager: HwApsManagerService, registerCallback, start !
09-09 13:33:57.136 13025 13025 D Hwaps   : APS: EventAnalyzed: registerCallbackInApsManagerService, mPkgName:app.revanced.manager.flutter; result = true
09-09 13:33:58.073 13025 13025 V AudioManager: querySoundEffectsEnabled...
09-09 13:33:58.076 13025 13025 D HwCustConnectivityManagerImpl: isBlockNetworkRequestByNonAis, INVALID_SUBSCRIPTION_ID
09-09 13:33:58.077 13025 13025 I ConnectivityManager: requestNetwork and the calling app is: app.revanced.manager.flutter
09-09 13:33:58.601 13025 14089 W manager.flutte: resources.arsc in APK '/data/app/com.huawei.docs-U9tQgLflNdi5lLISOmPZ6g==/base.apk' is compressed.
09-09 13:33:58.747 13025 14089 W manager.flutte: resources.arsc in APK '/data/app/com.apkpure.aegon-DovX3NnjKswwoCoeYpRx2w==/base.apk' is compressed.
09-09 13:33:58.890 13025 13025 W HiTouch_PressGestureDetector: Touch pointer move a lot. The moving distance of X is:26.975037, limit is:60The moving distance of Y is:67.97058, limit is:60
09-09 13:33:59.208 13025 13025 W HiTouch_PressGestureDetector: Touch pointer move a lot. The moving distance of X is:22.978699, limit is:60The moving distance of Y is:67.97058, limit is:60
09-09 13:33:59.742 13025 13025 W HiTouch_PressGestureDetector: Touch pointer move a lot. The moving distance of X is:5.994446, limit is:60The moving distance of Y is:105.95422, limit is:60
09-09 13:34:00.066 13025 13025 W HiTouch_PressGestureDetector: Touch pointer move a lot. The moving distance of X is:41.96118, limit is:60The moving distance of Y is:68.97009, limit is:60
09-09 13:34:00.523 13025 13025 I AwareBitmapCacher: init lrucache size: 2097152 pid=13025
09-09 13:34:02.727 13025 13025 W HiTouch_PressGestureDetector: Touch pointer move a lot. The moving distance of X is:9.990784, limit is:60The moving distance of Y is:68.970215, limit is:60
09-09 13:34:03.070 13025 13025 W HiTouch_PressGestureDetector: Touch pointer move a lot. The moving distance of X is:48.95465, limit is:60The moving distance of Y is:67.97058, limit is:60
09-09 13:34:04.096 13025 13025 W HiTouch_PressGestureDetector: Touch pointer move a lot. The moving distance of X is:12.987976, limit is:60The moving distance of Y is:74.96753, limit is:60
09-09 13:34:04.535 13025 13025 W HiTouch_PressGestureDetector: Touch pointer move a lot. The moving distance of X is:20.98059, limit is:60The moving distance of Y is:65.971436, limit is:60
09-09 13:34:05.057 13025 13025 W HiTouch_PressGestureDetector: Touch pointer move a lot. The moving distance of X is:27.974121, limit is:60The moving distance of Y is:74.96759, limit is:60
09-09 13:34:05.957 13025 13025 V AudioManager: querySoundEffectsEnabled...
09-09 13:34:06.067 13025 13025 D DecorView: showOrHideHighlightView: hasFocus=false; winMode=1; isMrgNull=true
09-09 13:34:06.269 13025 13025 I RmeSchedManager: init Rme, version is: v1.0
09-09 13:34:06.520 13025 13466 D ZrHung.AppEyeUiProbe: not watching, wait.
09-09 13:34:12.945 13025 13481 W libEGL  : EGLNativeWindowType 0x79854f7010 disconnect failed
09-09 13:34:12.949 13025 13475 W libEGL  : EGLNativeWindowType 0x797c8b4750 disconnect failed
09-09 13:34:17.618 13025 13480 W DisplayEventDispatcher: dispatcher 0x797864da40 ~ ignoring unknown event type 0x736f6674
09-09 13:34:17.619 13025 13025 W DisplayEventDispatcher: dispatcher 0x798552a140 ~ ignoring unknown event type 0x736f6674
09-09 13:34:18.237 13025 13025 D ActivityThread: Won't deliver top position change in state=5
09-09 13:34:18.241 13025 13466 D ZrHung.AppEyeUiProbe: restart watching
09-09 13:34:18.242 13025 13025 I RmeSchedManager: init Rme, version is: v1.0
09-09 13:34:18.242 13025 13025 I RtgSchedEvent: current pid:13025 AppType:-1
09-09 13:34:18.244 13025 13025 I DecorView[]: pkgName:app.revanced.manager.flutter old windowMode:1 new windoMode:1, isFixedSize:false
09-09 13:34:18.253 13025 13475 D mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
09-09 13:34:18.258 13025 13481 D mali_winsys: EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
09-09 13:34:18.281 13025 13025 I HwViewRootImpl: removeInvalidNode all the node in jank list is out of time
09-09 13:34:18.282 13025 13025 D DecorView: showOrHideHighlightView: hasFocus=true; winMode=1; isMrgNull=true
09-09 13:34:18.282 13025 13025 W InputMethodManager: startInputReason = 1
09-09 13:34:18.282 13025 13025 W HwRemoteInputMethodManager: isCasting false because IHwDistributedWindowManager is invalid.
09-09 13:34:20.047 13025 13025 V AudioManager: querySoundEffectsEnabled...
```


### Patcher logs

```shell
Copying APK
Reading APK
Decoding app manifest
Loading patches
Merging integrations
Deleting existing resource cache directory
Decoding resources
Executing patches
Alternative thumbnails succeeded
Always autorepeat succeeded
Client spoof succeeded
Comments succeeded
Copy video url succeeded
Custom branding succeeded
Custom player overlay opacity succeeded
Disable Shorts on startup succeeded
Disable auto captions succeeded
Disable player popup panels succeeded
Disable zoom haptics succeeded
Enable tablet layout succeeded
External downloads succeeded
HDR auto brightness succeeded
Hide Shorts components succeeded
Hide ads succeeded
Hide album cards succeeded
Hide autoplay button succeeded
Hide breaking news shelf succeeded
Hide captions button succeeded
Hide cast button succeeded
Hide crowdfunding box succeeded
Hide email address succeeded
Hide endscreen cards succeeded
Hide filter bar succeeded
Hide floating microphone button succeeded
Hide info cards succeeded
Hide layout components succeeded
Hide load more button succeeded
Hide player buttons succeeded
Hide seekbar succeeded
Hide timestamp succeeded
Hide video action buttons succeeded
Hide watermark succeeded
Minimized playback succeeded
Navigation buttons succeeded
Old video quality menu succeeded
Open links externally succeeded
Playback speed succeeded
Player flyout menu succeeded
Premium heading succeeded
Remember video quality succeeded
Remove player controls background succeeded
Return YouTube Dislike succeeded
Seekbar tapping succeeded
SponsorBlock succeeded
Spoof app version succeeded
Swipe controls succeeded
Tablet mini player succeeded
Theme succeeded
Vanced MicroG support succeeded
Video ads succeeded
Wide searchbar succeeded
Compiling modified dex files
Compiling modified resources
Signing APK
Patched
```


### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

--- a/android/app/src/main/kotlin/app/revanced/manager/flutter/MainActivity.kt
+++ b/android/app/src/main/kotlin/app/revanced/manager/flutter/MainActivity.kt
@@ -384,13 +384,7 @@ class MainActivity : FlutterActivity() {
 
                 updateProgress(0.9, "Signing...", "Signing APK")
 
-                try {
-                    Signer("ReVanced", keystorePassword)
-                        .signApk(patchedFile, outFile, keyStoreFile)
-                } catch (e: Exception) {
-                    print("Error signing APK: ${e.message}")
-                    e.printStackTrace()
-                }
+                Signer("ReVanced", keystorePassword).signApk(patchedFile, outFile, keyStoreFile)
 
                 updateProgress(1.0, "Patched", "Patched")
             } catch (ex: Throwable) {

